actions/src/jupyter.py

```diff
@@ -29,18 +30,21 @@ def user_delete(
         Delete users from the given environment
         """
         jupyter_keys = self.config["jupyter"]
-
-        key_name = None
-        env = jupyter_env.casefold()
-        if env == "prod".casefold():
-            key_name = "prod_token"
-        if env == "training".casefold():
-            key_name = "training_token"
-        if env == "dev".casefold():
-            key_name = "dev_token"
-        if not key_name:
-            raise KeyError("Unknown Jupyter environment")
```

Comment:
:+1: Good change, this is a faux-par on my part considering I bang on about not putting logic in the actions layer....